lib/cartopy/crs.py

```diff
@@ -1310,12 +1310,27 @@ def y_limits(self):
 class Stereographic(Projection):
     def __init__(self, central_latitude=0.0, central_longitude=0.0,
                  false_easting=0.0, false_northing=0.0,
-                 true_scale_latitude=None, globe=None):
+                 true_scale_latitude=None,
+                 scale_factor=None, globe=None):
         proj4_params = [('proj', 'stere'), ('lat_0', central_latitude),
                         ('lon_0', central_longitude),
                         ('x_0', false_easting), ('y_0', false_northing)]
-        if true_scale_latitude:
+
+        if true_scale_latitude is not None:
+            if central_latitude not in (-90., 90.):
+                warnings.warn('"true_scale_latitude" parameter is only used '
+                              'for polar stereographic projections. Consider '
+                              'the use of "scale_factor" instead.')
             proj4_params.append(('lat_ts', true_scale_latitude))
+
+        if scale_factor:
```

Comment:
I'd be more comfortable if we checked None explicitly. (I know ``scale_factor=0`` doesn't really make any sense, but I'd rather not ignore it silently.)

Reply:
Sure.